doc/intel-gpu-dkms.rst

```diff
@@ -63,10 +63,10 @@ version used below to build DKMS packages) and its headers::
   sudo apt-get update
 
   # for Ubuntu 20.04
-  sudo apt-get install linux-headers-5.14.0-1045-oem linux-image-unsigned-5.14.0-1045-oem
+  sudo apt-get install linux-headers-5.14.0-1054-oem linux-image-unsigned-5.14.0-1054-oem
```

Comment:
Where 1054 and 1020 came from? Was that something latest which you've tried out?

I am asking because this file implies which versions of -oem kernel were tested by DKMS team: https://github.com/intel-gpu/intel-gpu-i915-backports/blob/ubuntu/main/versions, and they claim 1047 and 1014. It does not mean that 1054/1020 won't work (in fact they should), but I would like to keep consistency with DKMS projects.

Reply:
Yes, I had tried this on both 1054 and 1020 and they work fine. However, I can help to change it back to 1047 and 1014 to be consistent with DKMS